playground/backend/internal/executors/executor_go.go

```diff
@@ -0,0 +1,42 @@
+package executors
+
+import (
+	"beam.apache.org/playground/backend/internal/fs_tool"
+	"context"
+	"os/exec"
+	"strings"
+)
+
+const (
+	goCompileCmd = "go"
+	goTestCmd    = "go"
+)
+
+var goCompileArgs = []string{"build", "-o", "bin"}
+var goTestArgs = []string{"test", "-v"}
+
+func getGoCompileCmd(ctx context.Context, paths *fs_tool.LifeCyclePaths) (*exec.Cmd, error) {
```

Comment:
@TSultanov I remember we discussed having an interface with different implementations for different languages. We are not doing it here right?

Reply:
Implemented interfaces in new iteration of these changes